web-client/src/presenter/computeds/formattedTrialSessionDetailsForFilteredEligibleCases.js

```diff
@@ -0,0 +1,31 @@
+import { formattedTrialSessionDetails } from './formattedTrialSessionDetails';
+import { state } from 'cerebral';
+
+export const formattedTrialSessionDetailsForFilteredEligibleCases = (
+  get,
+  applicationContext,
+) => {
+  const trialSessionDetails = formattedTrialSessionDetails(
```

Comment:
🔧 Instead of re-computing this value, I believe it can be retrieved off of state!

Reply:
The thought process was to not alter the original computed (formattedTrialSessionDetails) since it is used elsewhere and therefore may break other portions of the app.  So, the filter was split out into its own computed and calls the formattedTrialSessionDetails because the filter is only needed on this view.  If you just pull it off of state, the filter no longer works.  I am not sure how else to do this.  I know this is a little weird, calling one computed from another.